### DISCUSSION TITLE
Problem with sending the application

### DISCUSSION THREAD

**Discussion by k4rdel** [community/community#127404] [GitHub Education] [answered]
### Hi everyone,

Hello, I'm trying to send my school certificate/school ID again to confirm my github student application, but every time I upload the photos and click the "process my application" button, nothing happens, the page refreshes and nothing happens. If I go to check my application, it takes me back to this page and tells me to send it, I send photos and it's still the same. what can I do?

**Comment by davevad93** [comment, marked as answer]
Hi @k4rdel ,

I've seen a lot of people experiencing similar issues, check out this #126848 and this #126276 discussion, maybe they can be helpful.
